repository/src/main/java/org/apache/atlas/repository/store/graph/v1/DeleteHandlerV1.java

```diff
@@ -998,15 +998,24 @@ protected void deleteVertex(AtlasVertex instanceVertex, boolean force) throws At
 
         // Delete external references to this vertex - incoming edges from lineage or glossary term edges
         final Iterable<AtlasEdge> incomingEdges    = instanceVertex.getEdges(AtlasEdgeDirection.IN);
+        final Iterable<AtlasEdge> outgoingEdges    = instanceVertex.getEdges(AtlasEdgeDirection.OUT);
         final boolean             isPurgeRequested = RequestContext.get().isPurgeRequested();
 
+        if (!RequestContext.get().getDeleteType().equals(DeleteType.SOFT)) {
+            for (AtlasEdge edge : outgoingEdges) {
+                if (isRelationshipEdge(edge))
+                    AtlasRelationshipStoreV2.saveRelationshipDeletionContext(RequestContext.get().getDeleteType(), null, edge, edge.getOutVertex(), edge.getInVertex(), entityRetriever);
```

Comment:
Lets not pass `null` as arguments, rather overload the method `saveRelationshipDeletionContext`

Reply:
Done